hasher-matcher-actioner/terraform/api/variables.tf

```diff
@@ -136,3 +136,9 @@ variable "partner_image_buckets" {
     params = map(string)
   }))
 }
+
+variable "integration_api_access_token" {
```

Comment:
Can users disable access via access token if they only want to use a cognito pool?

Maybe set a default here of empty string and in the auth lambda check for empty string before allowing?

Reply:
yes, that happens automatically based on my testing but incase it is system dependent (i've been burned by tf null checks before) I add that logic explicitly.